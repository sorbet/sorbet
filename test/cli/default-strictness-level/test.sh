#!/usr/bin/env bash

main/sorbet --silence-dev-message --default-strictness-level=true test/cli/default-strictness-level/default-strictness-level.rb 2>&1
