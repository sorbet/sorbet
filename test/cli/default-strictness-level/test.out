test/cli/default-strictness-level/default-strictness-level.rb:4: Method `bar` does not exist on `Foo` https://srb.help/7003
     4 |Foo.new.bar
                ^^^
  Got `Foo` originating from:
    test/cli/default-strictness-level/default-strictness-level.rb:4:
     4 |Foo.new.bar
        ^^^^^^^
Errors: 1
